Readme.md

```diff
@@ -22,3 +22,27 @@ To contact the steering committee:
 
 -   On GitHub: `@slsa-framework/slsa-steering-committee`
 -   Via email: slsa-steering-committee@googlegroups.com
+
+## Special Interest Groups
+If you would like to participate in a SIG, come join a meeting or reach out directly via slack/email (see information below)!
+
+[Specification](9._SIG_Charter_-_Specification.md)
+-   [Mark Lodato](https://github.com/MarkLodato) - Google
+-   [Joshua Lock](https://github.com/joshuagl) - VMware
+-   Via slack: [#slsa-specification](https://openssf.slack.com/archives/C03NUSAPKC6)
```

Comment:
As per https://github.com/slsa-framework/slsa/issues/499, this link fails if the user does not already have an account. Unfortunately we need to write this out as `#slsa-specification on https://slack.openssf.org`.

Same below.